packages/custom-elements/src/CustomElementInternals.ts

```diff
@@ -395,17 +360,13 @@ export default class CustomElementInternals {
    * Runs HTML's 'look up a custom element definition', excluding the namespace
    * check.
    *
-   * @private
-   * @param {!Document} doc
-   * @param {string} localName
-   * @return {!CustomElementDefinition|undefined}
```

Comment:
I think we should keep explicit return types where they exist. I'd say that this applies more strongly to the wrappers in `src/Patch`, but you were able to remove the `Utilities.setPropertyUnchecked`, which was basically just there to blindfold Closure so it would stop complaining about modifying globals. Is TS enforcing correct signatures for functions that overwrite globals now? Basically, if the native signature is enforced on the wrappers in `src/Patch`, then awesome let's leave them implicit. If not, we should explicitly annotate the whole signature.

Reply:
Yep, the assignments to things like `Document.prototype.importNode` are type checked, which ensured that our patched version has the same type as the native version.